vitest.config.ts

```diff
@@ -22,5 +21,21 @@ export default defineConfig({
     outputFile: {
       junit: 'test-results/junit.xml',
     },
+    projects: [
+      {
+        extends: true,
+        test: {
+          name: 'unit',
+          include: ['src/**/*.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
```

Comment:
Can we do this or will it match something we don't want?

```suggestion
          include: ['src/**/*.test.*'],
```

Reply:
```
cd src
find . | sort | grep '\.test\.' > out.txt
```

The ending extensions I see are
```
.test.tsx
.test.ts
.test.json
```